addons/website_forum/views/forum_forum_templates_layout.xml

```diff
@@ -141,15 +140,27 @@
                     <t t-else="">
                         <t t-if="forum.mode == 'questions'">
                             <a t-attf-href="?#{ keep_query('search', 'sorting', 'my', filters='solved') }"
-                               class="dropdown-item">Solved
+                                class="dropdown-item">Solved
```

Comment:
The comment about this seems to have been lost. I intently aligned on the t of t-attf-href ;) .